### PR TITLE
feat(mantine): add xml language support to the CodeEditor

### PR DESCRIPTION
**Pull request by gdostie**
### Proposed Changes

Adding syntax highlighting and formatter for the xml language.

### Potential Breaking Changes

None

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/website/src/examples/form/code-editor/CodeEditorXML.demo.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import {CodeEditor, useForm} from '@coveord/plasma-mantine';
+
+export default () => {
+    const form = useForm({
+        initialValues: {
```
Comment (aiheon) [inline packages/website/src/examples/form/code-editor/CodeEditorXML.demo.tsx:5]: having the code for the demo is nice but you forgot to plug it in the actual page 😆

Comment (gdostie) [reply packages/website/src/examples/form/code-editor/CodeEditorXML.demo.tsx:5]: I didn't forget 😉 Its just that having multiple examples with the new sandbox isn't supported yet. It is in my upcoming PR :hype:. I plan on updating the code editor examples after this particular upcoming PR is merged

Comment (aiheon) [reply packages/website/src/examples/form/code-editor/CodeEditorXML.demo.tsx:5]: ohhh noice ❤️‍🔥  I was too eager to see the XML in action it seems 😂

**Review by aiheon — APPROVED** [review]

**Review by SimonMilord — APPROVED** [review]
Good Job!

**Pull request merged**